.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
```

Comment:
Is that needed/worth to have?

Reply:
That's just a comment pointing to the documentation. It was in the template I copied from, I can remove it, IDC.